### ISSUE TITLE
Validate::isEmail returns true on null

### ISSUE THREAD

**Issue by seigieu** [thirtybees/thirtybees#1447]
Example test code: 

```
var_dump(Validate::isEmail(null)); // true << Wrong one?
var_dump(Validate::isEmail('')); // false
var_dump(Validate::isEmail('test')); // false
var_dump(Validate::isEmail('test@test.pl')); //true
var_dump(Validate::isEmail(false)); //false
```
possible (simplest one) solution
```
 public static function isEmail($email)
    {
        if(empty($email)) return false; // HERE
        // Convert to IDN first if necessary
        if (mb_detect_encoding($email, 'UTF-8', true) && mb_strpos($email, '@') > -1) {
            // Convert to IDN
            list ($local, $domain) = explode('@', $email, 2);
            $domain = Tools::utf8ToIdn($domain);
            $email = "$local@$domain";
        }

        try {
            \Swift_Message::newInstance()->setFrom($email);

            return true;
        } catch (\Exception $e) {
            return false;
        }
    }
```

**Comment by seigieu** [comment]
PS: If someone needs, here is override

```
class Validate extends ValidateCore
{
    public static function isEmail($email)
    {
        if(empty($email)) return false;
        return parent::isEmail($email);
    }
}
```

**Comment by getdatakick** [comment]
This is already fixed in main branch, commit e62652532b5a910fa3faabd557d60bb075f84cb1

**Comment by seigieu** [comment]
Ok. Searched only in issues.
